src/main/java/apoc/metrics/Metrics.java

```diff
@@ -204,4 +302,24 @@ private final double doubleFrom(LoadCsv.CSVResult r, String column) {
     private final long longFrom(LoadCsv.CSVResult r, String column) {
         return Long.parseLong(r.map.get(column).toString());
     }
+
+
+    // This is the list of dbms.directories.* valid configuration items for neo4j.
+    // https://neo4j.com/docs/operations-manual/current/reference/configuration-settings/
+    // Usually these reside under the same root but because they're separately configurable, in the worst case
+    // every one is on a different device.
+    //
+    // More likely, they'll be largely similar metrics.
+    public static final String [] neo4jDirectoryConfigurationSettingNames = new String [] {
```

Comment:
constants should be NEO4J_DIRECTORY_CONFIGURATION_SETTING_NAMES
use Arrays.asList(...) then you can use contains / stream directly on it.